components/pricing_modal/self_hosted_content.tsx

```diff
@@ -0,0 +1,241 @@
+                        price='$10'
+                        rate={formatMessage({id: 'pricing_modal.rate.userPerMonth', defaultMessage: '/user/month'})}
+                        briefing={{
+                            title: formatMessage({id: 'pricing_modal.briefing.title', defaultMessage: 'Top features'}),
+                            items: SelfHostedBriefing.professional,
+                        }}
+                        extraBriefing={{
+                            title: formatMessage({id: 'pricing_modal.extra_briefing.title', defaultMessage: 'More features'}),
+                            items: SelfHostedExtraBriefing.professional,
+                        }}
+                        buttonDetails={{
+                            action: () => {
+                                window.open(CloudLinks.SELF_HOSTED_LOGIN, '_blank');
+                            },
+                            text: formatMessage({id: 'pricing_modal.btn.upgrade', defaultMessage: 'Upgrade'}),
+                            disabled: !isAdmin,
+                            customClass: isPostSelfHostedEnterpriseTrial ? ButtonCustomiserClasses.special : ButtonCustomiserClasses.active,
+                        }}
+                        planLabel={
+                            <PlanLabel
+                                text={formatMessage({id: 'pricing_modal.planLabel.mostPopular', defaultMessage: 'MOST POPULAR'})}
+                                bgColor='var(--title-color-indigo-500)'
+                                color='var(--button-color)'
+                                firstSvg={<StarMarkSvg/>}
+                                secondSvg={<StarMarkSvg/>}
+                            />}
+                    />
+                    <Card
+                        id='enterprise'
+                        topColor='var(--denim-button-bg)'
+                        plan='Enterprise'
+                        price={formatMessage({id: 'pricing_modal.price.contactSales', defaultMessage: 'Contact sales'})}
+                        briefing={{
+                            title: formatMessage({id: 'pricing_modal.briefing.title', defaultMessage: 'Top features'}),
+                            items: SelfHostedBriefing.enterprise,
+                        }}
+                        extraBriefing={{
+                            title: formatMessage({id: 'pricing_modal.extra_briefing.title', defaultMessage: 'More features'}),
+                            items: SelfHostedExtraBriefing.enterprise,
+                        }}
+                        planExtraInformation={(isPostSelfHostedEnterpriseTrial || !isAdmin) ? undefined : <ContactSalesCTA/>}
+                        buttonDetails={(isPostSelfHostedEnterpriseTrial || !isAdmin) ? {
+                            action: () => {
+                                trackEvent('self_hosted_pricing', 'click_enterprise_contact_sales');
+                                window.open(LicenseLinks.CONTACT_SALES, '_blank');
```

Comment:
@AGMETEOR in the PR spinwick for self-managed it seems that it is trying to open the cloud contact sales https://customers.mattermost.com/cloud/contact-us instead of: `LicenseLinks.CONTACT_SALES` -> https://mattermost.com/contact-sales/

Reply:
🤔